.github/workflows/acceptance_tests_secondary.yml

```
@@ -6,6 +6,7 @@ on:
       - 'web/html/src/**'
       - 'testsuite/**'
       - '.github/workflows/acceptance_tests_secondary.yml'
+      - '!java/*.changes'
```

Existing changelog file is named `spacewalk-java.changes` but the new files going to be named like: `spacewalk-java.changes.cbbayburt.myfeature`.

So when PRs are changed, the entries must be moved from `spacewalk-java.changes` to the new files, so both files will be modified.

To cover both, I think it should be something like this:
```suggestion
      - '!java/*.changes*'
```